src/main/java/org/isf/patient/service/PatientIoOperations.java

```diff
@@ -239,4 +280,21 @@ public boolean isCodePresent(Integer code) throws OHServiceException {
 		return repository.existsById(code);
 	}
 
+	public PatientProfilePhoto retrievePatientProfilePhoto(Patient patient) {
+		boolean isLoadProfilePhotoFromDb = LOAD_FROM_DB.equals(GeneralData.PATIENTPHOTOSTORAGE);
+		if (isLoadProfilePhotoFromDb) {
+			Hibernate.initialize(patient.getPatientProfilePhoto());
+			return patient.getPatientProfilePhoto();
+		} else {
+			try {
+				fileSystemPatientPhotoRepository.loadInPatient(patient, GeneralData.PATIENTPHOTOSTORAGE);
+				return patient.getPatientProfilePhoto();
+			} catch (OHServiceException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
```

Comment:
Shouldn't this be a `log` message instead of a `printStackTrace()`?

Reply:
Actually the service should only throw OHServiceException, I'll fix that, thanks!